Cargo.toml

```diff
@@ -566,6 +568,9 @@ signature = "2.1.0"
 sec1 = "0.7.0"
 parking_lot = "0.12.0"
 paste = "1.0.7"
+passkey-authenticator = { git = "https://github.com/aptos-labs/passkey-rs", branch = "fix-passkey-rs", features = ["testable"] }
+passkey-client = { git = "https://github.com/aptos-labs/passkey-rs", branch = "fix-passkey-rs" }
+passkey-types = { git = "https://github.com/aptos-labs/passkey-rs", branch = "fix-passkey-rs" }
```

Comment:
should pin to a commit?

Reply:
Yes you're right. I've merged the changes from my `passkey-rs` fork to the upstream branch so I can change the git url to the 1password `passkey-rs` url and will pin to commit. 

They're doing an audit of the package before releasing a new version. Once they publish the package to cargo i'll change this to reference the new version.